.travis.yml

```diff
@@ -1,24 +1,22 @@
-sudo: required
-
 language: bash
+services: docker
 
-services:
-  - docker
-
-install:
-  - docker build -t quay.io/azavea/django:${TRAVIS_COMMIT:0:7} .
+env:
+  - VERSION=1.9 PYTHON_VERSION=2.7 PG_MAJOR=9.5 PG_VERSION=9.4+165+deb8u2 VARIANT=slim
```

Comment:
The main reason I did exact version constraints here was due to how PostgreSQL dependency versions are handled in the upstream Docker images: https://github.com/docker-library/postgres/blob/5159417968c6a08e2ed784498cba28f22a74b03e/9.6/Dockerfile#L40

I think it would be valuable to see how bad the maintenance overhead is before switching to globs.